### ISSUE TITLE
Gallbladder palpable on exam

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10046]
**Preferred term label:**
Gallbladder palpable on exam

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Porcelain gallbladder

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am going to make a term called Porcelain gallbladder.
I am not sure that palpable is the major point, see " If the patient is very thin or if the gallbladder is enlarged, a firm, hard gallbladder can be palpated in the right upper quadrant of the abdomen."
https://www.ncbi.nlm.nih.gov/books/NBK518979/

Adding new term: 
- ID: HP:6000451
- label: Porcelain gallbladder
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:30085521
- def: Porcelain gallbladder refers to the condition in which the inner gallbladder wall is encrusted with calcium. The wall becomes brittle, hard, and often takes on a bluish hue. It is usually found incidentally on plain abdominal x-rays or other imaging because most patients are asymptomatic.
- comment: Most cases of porcelain gallbladder are asymptomatic. However, some patients do exhibit signs of chronic cholecystitis. Patients with chronic cholecystitis usually present with dull right upper abdominal pain that radiates to the mid back or right scapular tip. It is usually associated with fatty food ingestion. The classic physical examination will demonstrate right upper abdominal pain with deep palpation (Murphy's sign). If the patient is very thin or if the gallbladder is enlarged, a firm, hard gallbladder can be palpated in the right upper quadrant of the abdomen.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10046
- synonyms: Calcified gallbladder(NONE)|Calcifying cholecystitis(NONE)|Cholecystopathia chronica calcarea(NONE)

**Comment by MickeySegal** [comment]
That describes one subset of palpable gallbladder.  but we now have a second instance: https://www.ncbi.nlm.nih.gov/books/NBK448112/
" A lump may be palpable deep to the fistula due to the underlying inflamed or malignant gallbladder."
So it seems worthwhile having the more general term too.
